src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -100,6 +104,11 @@
     private OCFile mFile = null;
     private FileListListAdapter mAdapter;
 
+    private LinearLayout uploadFilesLinearLayout;
```

Comment:
These four variables do not need to be declared globally, you can declare it directly in registerFabUploadListeners method, where are going to be used.